src/config.js

```diff
@@ -0,0 +1,6 @@
+export {
```

Comment:
I'm sorry I should have clarified earlier. These need variables need to export values (See below).

Also the `const` keyword cannot be used inside an object. Either remove the `const` keyword and format the values as:  
```javascript
export default {
key: value,
key2: value
}
```
Or, format as:

```javascript
export const key = value;
export const key = value2;
```

- `CITY: 'Greensboro'`
- `CITY_NAME_ABBREVIATED: 'GSO'` 
- `COUNTY: 'Guilford'` 
- `APP_TITLE: GoVoteGSO`

Thank you so much for your help! Let me know if you have any other questions.

Schaffer